Bottom: e65b1735c8b77d515ec29478985323ecf74db060
Top:    1fe2deb24347daba4f4d727b8f66346536f12ad6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-22 01:18:22 +0000

draw.c: Refactor color setting code to be more compact


---

diff --git a/src/draw.c b/src/draw.c
index 32aacd5..ca29442 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -81,7 +81,6 @@ static bool doing_assy = false;
 static void DrawEverything (const BoxType *);
 static void DrawPPV (int group, const BoxType *);
 static void AddPart (void *);
-static void SetPVColor (PinType *, int);
 static void DrawEMark (ElementType *, Coord, Coord, bool);
 static void DrawRats (const BoxType *);
